routes/api/contacts.js

```diff
@@ -1,25 +1,160 @@
-const express = require('express')
+import express from 'express';
+import Joi from 'joi';
 
-const router = express.Router()
+import {
+  listContacts,
+  getContactById,
+  removeContact,
+  addContact,
+  updateContact,
+} from './../../models/contacts.js';
+
+export const router = express.Router();
+const sendErrorResponse = (res, status, message) => {
+  res.status(status).json({ message });
+};
+
+const schemaAdd = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().email().required(),
+  phone: Joi.string().required(),
+});
+
+const schemaEdit = Joi.object({
+  name: Joi.string(),
+  email: Joi.string().email(),
+  phone: Joi.string(),
+}).unknown(false);
 
 router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+  try {
+    const contacts = await listContacts();
+
+    if (!contacts) {
+      sendErrorResponse(res, 404, `Contact list not found`);
```

Comment:
return before `sendErrorResponse`

get rid of `return false` under please.

Reply:
actually you dont need this one. You should get always empty array or array of contacts.